huntsman/tests/test_observatory.py

```diff
@@ -0,0 +1,380 @@
+import os
+import pytest
+
+import time
+from astropy import units as u
+from astropy.time import Time
+
+from pocs import hardware
+import pocs.version
+from pocs.observatory import Observatory
+from pocs.scheduler.dispatch import Scheduler
+from pocs.scheduler.observation import Observation
+from pocs.camera import create_cameras_from_config
+from pocs.utils import error
+
+from huntsman.camera.pyro import Camera as PyroCamera
+
+
+@pytest.fixture
+def simulator():
+    """ We assume everything runs on a simulator
+
+    Tests that require real hardware should be marked with the appropriate
+    fixture (see `conftest.py`)
+    """
+    return hardware.get_all_names(without=['night'])
+
+
+@pytest.fixture
+def observatory(config, simulator):
+    """Return a valid Observatory instance with a specific config."""
+    cameras = create_cameras_from_config(config)
```

Comment:
Looks like this was supposed to change?

Reply:
Yes. This entire file is just copied verbatim from the POCS version right now. There will be a few things that needs changing, and other buts that are redundant with `test_huntsman.py`. Haven't had chance to go through it yet.